### DISCUSSION TITLE
AcceptAnyServerCertificate- clarification

### DISCUSSION THREAD

**Discussion by Henery309** [dotnet/yarp#965] [Q&A] [answered]
Looking through the documentations for DangerousAcceptAnyServerCertificate it says `indicates whether the server's SSL certificate validity is checked by the client`. What is server's SSL referring to in this statement? Is it the yarp destination clusters? 

Example - assuming my destination is example.com/api/v1 <-- If DangerousAcceptAnyServerCertificate = true then ssl certification will not be validated for example.com/api/v1?

**Comment by Tratcher** [comment, marked as answer]
Correct, the certificates in this situation are from the destination servers.
